api/members/model.ts

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+import {
+    Column,
+    CreateDateColumn,
+    Entity, EntityManager, FindManyOptions,
+    ManyToMany, JoinTable,
+    PrimaryGeneratedColumn,
+    TransactionManager,
+    UpdateDateColumn
+} from "typeorm";
+import EventModel from "../event/model";
+
+
+@Entity({name: 'members'})
+export default class MembersModel {
+    @PrimaryGeneratedColumn()
+    id: number;
+
+    @Column({length: 64})
+		firstName: string;
+		
+		@Column({length: 64})
+		lastName: string;
+
+		@Column({length: 140})
+		imageUrl: string;
```

Comment:
Why 140?

Reply:
paying homage to the original Tweet length

Reply:
Ah hahaha okay. Can you update it to 255 to match the database column length?

Can you also updated firstName and lastName to be 80 to match their database columns?